src/LibraryManager/Manifest.cs

```diff
@@ -63,6 +64,17 @@ public Manifest(IDependencies dependencies)
         [JsonConverter(typeof(LibraryStateTypeConverter))]
         public IEnumerable<ILibraryInstallationState> Libraries => _libraries;
 
+        /// <summary>
+        /// Validates libraries in this manifest
+        /// </summary>
+        /// <param name="libraries"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async Task<IEnumerable<ILibraryOperationResult>> ValidateLibrariesAsync(IEnumerable<ILibraryInstallationState> libraries, CancellationToken cancellationToken)
+        {
+            return await _librariesValidator.ValidateLibrariesAsync(libraries, cancellationToken).ConfigureAwait(false);
```

Comment:
>_librariesValidator [](start = 25, length = 19)

Does access of this need to be checked?

Reply:
You mean check for null? If so, then yes. 

---
In reply to: [195512000](https://github.com/aspnet/LibraryManager/pull/164#discussion_r195512000) [](ancestors = 195512000)